src/mame/drivers/gunpey.cpp

```diff
@@ -234,17 +234,21 @@ struct huffman_node_s
 	int arg1_val;
 };
 
+static inline u32 get_vmem_addr(int x, int y)
+{
+	return (x & 0x7ff) | ((y & 0x7ff) << 11);
+}
```

Comment:
Please leave the two blank lines between sections, and use `constexpr` for things that should be `constexpr` (also `inline` implies `static` in C++, there’s no need for both).  Also, why are the inputs signed when the output is unsigned?